ca_description/launch/create_description.launch

```diff
@@ -7,6 +7,7 @@
   <arg name="model" value="$(find ca_description)/urdf/create_2.xacro" doc="Path to Xacro description of the robot"/>
 
   <arg name="visualize" default="$(optenv VISUALIZE false)" doc="Flag to visualize rays in Gazebo"/>
+  <arg name="enable_roscontrol" default="$(optenv ROSCONTROL False)" doc="Flag to use roscontrol Gazebo plugin"/>
```

Comment:
Use `ROS_CONTROL` and put it in `true` by default.

```suggestion
  <arg name="enable_ros_control" default="$(optenv ROS_CONTROL true)" doc="Flag to use ros_control."/>
```

I also suggested a modification in `doc` because this flag can also be used with the real robot.

Reply:
Mmm, I don't agree with this...
The cmd_vel topic that this exposes doesn't quite match with the existing one, and can't be remapped.
Making move_base work with this will require a little extra work

Reply:
Ok, but address the rest of the sentence.

What's the difference?